Hangman Game/README.md

```diff
@@ -0,0 +1,15 @@
+# Package/Script Name
```

Comment:
Your README should look something like [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Reddit-Meme-Scraper/README.md). Please make appropriate changes.

Reply:
> Your README should look something like [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Reddit-Meme-Scraper/README.md). Please make appropriate changes.

okk @atarax665 now you see and plss review it and merged it